Makefile

```diff
@@ -1,5 +1,5 @@
 VERSION = 0.2.0
-RELEASE = 1
+RELEASE = 2
```

Comment:
If I'm not wrong, this should be reserved only for PRs that release a new version

Reply:
Understood. Will revert here and in the `.spec` file

Reply:
Done